frontend/src/components/notification/notification.tsx

```diff
@@ -0,0 +1,52 @@
+import React, {useEffect} from 'react';
+import {Alert, Snackbar} from "@mui/material";
+import CheckIcon from '@mui/icons-material/Check';
+import {useAppDispatch, useAppSelector} from "../../stores/redux/hooks";
+import {selectNotification, showAlert} from "../../stores/redux/notification/notificationSlice";
+import {INotification} from "../../types/notification";
+import {NotificationType} from "../../types/notificationType";
+
+export const createNotification = (message: string): INotification => {
```

Comment:
It is strange that you have function only for one successful case. Either create for all cases or create object in place